e2e/cypress/integration/enterprise/cloud/Billing/company_Information_spec.js

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Please use lower in file name.

Reply:
done , renamed file name with all lower case.

Reply:
It seems the file was duplicated in this PR. Also, can you update the folder name to use lower case as well?